server/routerlicious/packages/lambdas-driver/src/document-router/documentContext.ts

```diff
@@ -109,4 +109,12 @@ export class DocumentContext extends EventEmitter implements IContext {
 	public getContextError() {
 		return this.contextError;
 	}
+
+	public pause(offset: number, reason?: any) {
+		this.emit("pause", offset, reason); // TODO add listeners to pause the partition for circuit breaker functionality
+	}
+
+	public resume() {
+		this.emit("resume"); // TODO add listeners to pause the partition for circuit breaker functionality
```

Comment:
This is documentContext for document lambdas, I havent added event handlers for these yet. Will add these separately after rolling it out for scriptorium.

Reply:
Should these TODO comments be in the document lambdas, rather than in the context? Also, if these are tracked in a work item somewhere, there probably don't need to be TODO comments, or the comments should reference the work item. Many TODOs in the FF repo reference ADO work items like `// TODO: AB#1234 blah blah blah`

Reply:
Removed these comments and added a WI here https://dev.azure.com/fluidframework/internal/_workitems/edit/19992